modules/k6/k6.go

```diff
@@ -49,13 +54,27 @@ func SetEnvVar(variable string, value string) testcontainers.CustomizeRequestOpt
 	}
 }
 
-// WithCache uses the given volume as a cache for building the k6 binary.
-// If the volume does not exists, it is created.
-func WithCache(cache string) testcontainers.CustomizeRequestOption {
+// WithCache sets a volume as a cache for building the k6 binary
+// If a volume name is provided in the TC_K6_BUILD_CACHE, this volume is used.
+// If no value is provided, a volume is created and removed when the test session ends.
+func WithCache() testcontainers.CustomizeRequestOption {
+	var volOptions *mount.VolumeOptions
+
+	cacheVol := os.Getenv("TC_K6_BUILD_CACHE")
```

Comment:
Out of curiosity: it does not collide on two different test sessions, right? The first one creates the volume, and the second simply uses it. At least this is how `docker volume create` works.

Reply:
I'm not sure what you mean by colliding on two test sessions. Effectively, the idea is that the first session (`go test` command)  using this environment variable will create the volume and the subsequent invocations will use it.

Reply:
Yeah, this case:

1. the first `go test` creates the volume as it was passed as an env var, and it persists
2. the second `go test`. It will reuse the volume, right? Just want to double check that the second one does not fail because the volume already exists. I know it's handled by the docker client (we simply pass the volume mount to it), but want to verify it.